### PR TITLE
backport 'More powerfull logs for Neutron' for Quantum-3.2.1

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

**Comment by xenolog** [comment]
tested on Centos/HA

File: deployment/puppet/quantum/manifests/agents/dhcp.pp [hunk 1]
```diff
@@ -59,6 +58,11 @@
   quantum_dhcp_agent_config {
     'DEFAULT/debug':             value => $debug;
     'DEFAULT/verbose':           value => $verbose;
+    'DEFAULT/log_dir':          ensure => absent;
+    'DEFAULT/log_file':         ensure => absent;
+    'DEFAULT/log_config':       ensure => absent;
+    'DEFAULT/use_syslog':       ensure => absent;
+    'DEFAULT/use_stderr':       ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/quantum/manifests/agents/dhcp.pp:65]: it should be TRUE

File: deployment/puppet/quantum/manifests/agents/l3.pp [hunk 2]
```diff
@@ -47,6 +47,11 @@
   quantum_l3_agent_config {
     'DEFAULT/debug':          value => $debug;
     'DEFAULT/verbose':        value => $verbose;
+    'DEFAULT/log_dir':       ensure => absent;
+    'DEFAULT/log_file':      ensure => absent;
+    'DEFAULT/log_config':    ensure => absent;
+    'DEFAULT/use_syslog':    ensure => absent;
+    'DEFAULT/use_stderr':    ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/quantum/manifests/agents/l3.pp:54]: it should be TRUE

File: deployment/puppet/quantum/manifests/agents/metadata.pp [hunk 3]
```diff
@@ -22,6 +22,12 @@
 
   quantum_metadata_agent_config {
     'DEFAULT/debug':              value => $debug;
+    'DEFAULT/verbose':            value => $verbose;
+    'DEFAULT/log_dir':           ensure => absent;
+    'DEFAULT/log_file':          ensure => absent;
+    'DEFAULT/log_config':        ensure => absent;
+    'DEFAULT/use_syslog':        ensure => absent;
+    'DEFAULT/use_stderr':        ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/quantum/manifests/agents/metadata.pp:30]: it should be TRUE

File: deployment/puppet/quantum/manifests/plugins/ovs.pp [hunk 4]
```diff
@@ -63,14 +64,17 @@
...
     'OVS/tenant_network_type':      value  => $quantum_config['L2']['segmentation_type'];
     'OVS/enable_tunneling':         value  => $quantum_config['L2']['enable_tunneling'];
     'AGENT/polling_interval':       value  => $quantum_config['polling_interval'];
     'AGENT/root_helper':            value  => $quantum_config['root_helper'];
     'SECURITYGROUP/firewall_driver': value => 'quantum.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver';
+    'DEFAULT/log_dir':        ensure => absent;
+    'DEFAULT/log_file':       ensure => absent;
+    'DEFAULT/log_config':     ensure => absent;
+    'DEFAULT/use_syslog':     ensure => absent;
+    'DEFAULT/use_stderr':     ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/quantum/manifests/plugins/ovs.pp:77]: it should be TRUE

File: deployment/puppet/quantum/manifests/agents/dhcp.pp (hunk 1 above)
Comment (xenolog) [inline deployment/puppet/quantum/manifests/agents/dhcp.pp:65]: For our use-case:
this lines not allow in agent config. its will be inherits from quantum.conf

File: deployment/puppet/quantum/manifests/agents/l3.pp (hunk 2 above)
Comment (xenolog) [inline deployment/puppet/quantum/manifests/agents/l3.pp:54]: see upper

File: deployment/puppet/quantum/manifests/agents/metadata.pp (hunk 3 above)
Comment (xenolog) [inline deployment/puppet/quantum/manifests/agents/metadata.pp:30]: see upper

File: deployment/puppet/quantum/manifests/plugins/ovs.pp (hunk 4 above)
Comment (xenolog) [inline deployment/puppet/quantum/manifests/plugins/ovs.pp:77]: see upper

**Comment by bogdando** [comment]
+1

**Comment by monroid** [comment]
retest this please

**Comment by monroid** [comment]
retest this please

**Comment by xenolog** [comment]
retest please

**Comment by xenolog** [comment]
retest this please

**Comment by monroid** [comment]
retest this please

**Comment by aglarendil** [comment]
+2

**Pull request merged**